developer/engine/desktop/14.0/api/IKeymanBCP47Canonicalization/GetCanonicalTag.php

```diff
@@ -0,0 +1,29 @@
+<?php
+  require_once('includes/template.php');
+
+  head([
+    'title' => "IKeymanBCP47Canonicalization::GetCanonicalTag Method"
+  ]);
+?>
+
+<h1>IKeymanBCP47Canonicalization::GetCanonicalTag Method</h1>
+
+<h2>Introduction</h2>
+
+<p>The <code>IKeymanBCP47Canonicalization::GetCanonicalTag</code> method returns a canonical tag according to data from
+<a href="">SIL langtags.json.</a>. The method will convert ISO639-3 language tags ISO639-1 language tags, remove suppressed
```

Comment:
Would the link be https://github.com/silnrsi/langtags?
```suggestion
<a href="https://github.com/silnrsi/langtags">SIL langtags.json</a>. The method will convert ISO639-3 language tags ISO639-1 language tags, remove suppressed
```